### ISSUE TITLE
TypeScript errors since v7.0.4

### ISSUE THREAD

**Issue by gavvvr** [webdriverio-community/wdio-electron-service#810]
## Steps to reproduce

0. (Optional, for clean environment) `docker run --rm -it --entrypoint=sh node:22-alpine`
1. `npm init wdio@latest electron-e2e-ts`, choose Electron app testing + TypeScript
2. Add `typescript` to get `tsc` command: `cd electron-e2e-ts/ && npm i -D typescript`
3. Adjust `test/specs/test.e2e.ts` to add something like `await $('aria/Submit').click()` (some text editor might be needed in Docker, for example: `apk add --no-cache vim`)
4. run TS compiler (`npx tsc`) you'll see:

```
test/specs/test.e2e.ts:5:25 - error TS2339: Property 'click' does not exist on type 'Element | Promise<Element> | ChainablePromiseElementBase<Promise<Element>>'.
  Property 'click' does not exist on type 'Promise<Element>'.

5  await $('aria/Submit').click()
                          ~~~~~
```

Initially I noticed the problem in VSCode:
<img width="541" alt="image" src="https://github.com/user-attachments/assets/9cb4cb6b-74ea-4669-bf4d-805e84da11a7">

Roll back to v7.0.2 fixes the problem: `npm uninstall wdio-electron-service && npm i -D wdio-electron-service@7.0.2`. Now `npx tsc` and VSCode are happy.

**Comment by SaidbekAbdiganiev** [comment]
Do you have a tsconfig.json in the root of the test folder after installation? If not, create it and copy this config.

```
{
  "compilerOptions": {
    "types": ["@wdio/globals/types", "wdio-electron-service", "mocha"]
  },
  "include": ["**/*.ts"]
}
```

I remember it being somewhere in the examples, no idea why cli does not provide it by default

**Comment by gavvvr** [comment]
Hi @SaidbekAbdiganiev 
The wdio starter CLI includes correct (I believe) `tsconfig.json` and it includes the following:
```
    "types": [
      "node",
      "@wdio/globals/types",
      "expect-webdriverio",
      "@wdio/mocha-framework"
    ],
```

Unfortunately, tsc reports errors since v7.0.4

**Comment by goosewobbler** [comment]
I just ran the starter CLI and my tsconfig has the following types:

```
        "types": [
            "node",
            "@wdio/globals/types",
            "expect-webdriverio",
            "@wdio/mocha-framework",
            "wdio-electron-service"
        ],
```
I don't see anything in the changelog between 7.0.4 and 7.0.2 which might cause this type issue, but obviously something is wrong here.

**Comment by goosewobbler** [comment]
I found the cause, there are some commits missing from the changelog, presumably due to the release issues experienced around that time.  

https://github.com/webdriverio-community/wdio-electron-service/commit/d70ffccc9b7bce0d653d35f65fc9c34803d8252f#diff-11bc653c82fa9ebfed3bbdb52e23465a7c6524860d7ebc878d5695b3328fc47dR658

You can work around the issue by awaiting the result of $, e.g.

```(await $('aria/Submit')).click();```

**Comment by gavvvr** [comment]
If I do `await $('aria/Submit')`, then there is an issue raised by @typescript-eslint/await-thenable rule:

>Unexpected `await` of a non-Promise (non-"Thenable") value

I was wondering why the issue I posted gets reproduced in a clean project generated with `@wdio/cli`, but doesn't exist on my real project with the latest version of `wdio-electron-service`. While trying to understand the difference between clean project from @wdio-cli and my real-life project, I noticed the following: adding `import * as wdio from 'webdriverio'` also fixes the compiler error at `await $('aria/Submit').click()`. So at least one import `from 'webdriverio'` is needed and the problem with types disappears.
